tests/MockAdapter.ts

```diff
@@ -0,0 +1,10 @@
+import MockAdapter from "axios-mock-adapter";
+import { AxiosAdapter } from "../src/Adapters";
+
+export class AxiosMockAdapter extends AxiosAdapter {
+    public mockAdapter: MockAdapter
```

Comment:
Remove the public here to conform with the rest of the project. In TS it's implicitly public.

Reply:
added rule to eslint config for that